src/components/mobile-nav-popover/mobile-nav-popover.ts

```diff
@@ -36,14 +36,16 @@ export class MobileNavPopoverComponent {
   helpShow: boolean = false;
   assessmentId: any;
   assessmentShow: boolean = false;
+  userName: any;
 
 
   constructor(public navCtrl: NavController,
                     public navParams: NavParams,
                     public viewCtrl: ViewController,
                     public auth: AuthService) {
-    console.log('Hello MobileNavPopoverComponent Component');
     this.assessmentId = navParams.data.assessmentId;
+    this.userName = navParams.data.userName;
+    console.log(this.userName);
```

Comment:
```suggestion

```

Reply:
@dgonca is it possible to delete entire lines via these suggestions? We shall see....